log/log.go

```diff
@@ -302,3 +303,15 @@ func Fatalln(args ...interface{}) {
 func Fatalf(format string, args ...interface{}) {
 	baseLogger.sourced().Fatalf(format, args...)
 }
+
+type errorLogWriter struct{}
+
+func (errorLogWriter) Write(b []byte) (int, error) {
+	baseLogger.sourced().Error(string(b))
+	return len(b), nil
+}
+
+// ErrorLogger return log.Logger to be used as ErrorLog to catch HTTP server errors.
+func ErrorLogger() *log.Logger {
```

Comment:
Sorry for all the late comments. In order to be consistent with the golang naming, let's call it `NewErrorLogger`. The comment  needs to be updated as well then.